README.md

```diff
@@ -1,6 +1,8 @@
 # Wanda
 
+[![CI](https://github.com/trento-project/wanda/actions/workflows/ci.yaml/badge.svg)](https://github.com/fabriziosestito/rhai_rustler/actions/workflows/main.yaml)
```

Comment:
It would be nice to use `wanda` projects workflow file XD

Reply:
wopsi